index.html

```diff
@@ -8,310 +8,339 @@
+          </h3>
+          <ul class="list">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>Rspec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle57.png" alt="circle" /> Skills
+          </h3>
+          <ul class="list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="contact-form">
+      <div class="the-contact">
+        <h2 class="contact-p">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+
+        <form action="https://formspree.io/f/xnqwjqwy" method="post">
+          <div class="input__sec">
+            <div class="name">
+              <label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="name"
+                  placeholder="Full Name"
+                  required
+                  minlength="2"
+                  maxlength="30"
+                />
+              </label>
+            </div>
+            <p>
+              <label>
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email address"
+                  required
+                />
+              </label>
+            </p>
+            <label>
+              <textarea
+                name="textarea"
                 required
-                maxlength="30"
-                placeholder="Your Email"
-            /></label>
-            <textarea
-              id="message"
-              cols="30"
-              rows="10"
-              maxlength="500"
-              placeholder="Write your message here"
-              required
-            ></textarea>
-            <div class="submit-button">
-              <button type="submit">Get in touch</button>
+                minlength="2"
+                maxlength="500"
+                placeholder="Hello Vistor! Glad to see you!"
+              ></textarea>
+            </label>
+            <div class="btn__container">
+              <button type="submit" name="submit" class="btn__submit">
+                Get in touch
+              </button>
             </div>
-          </form>
-        </div>
-      </section>
-    </main>
+          </div>
+        </form>
+      </div>
+      <div class="hr1">
+        <hr class="hr" />
+      </div>
+      <div class="last-contact">
+        <ul class="footer-contacts">
+          <li>
+            <a href="https://github.com/mesfin345"
+              ><img src="icons/github.svg" alt="github addres"
+            /></a>
+          </li>
```

Comment:
- [ ] As requested by the previous reviewer, pleas make sure that the color of icons match with the icons provided in Figma.
In case if you do not know how to change the color: 

1. Open your github.svg in your editor.
2. look for `fill="#DBD8D7e`  and replace it with `fill= "#505F79"`

 eg:

```
<svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M17.9989 7.46114C17.9989 6.32124 17.6329 5.25043 16.9342 4.31779C17.2004 3.41969 17.2669 2.27979 17.2004 0.967185C17.1671 0.414508 16.7345 0 16.2022 0C15.9028 0 13.3741 0.0345423 11.9101 1.38169C10.6458 1.1399 9.31495 1.1399 8.01736 1.38169C6.58667 0.0345423 4.05802 0 3.7253 0C3.19296 0 2.76042 0.414508 2.72715 0.967185C2.62734 2.27979 2.72715 3.41969 2.99333 4.31779C2.29462 5.28497 1.92863 6.35579 1.92863 7.46114C1.92863 9.8791 3.7253 12.0553 6.45358 13.0915C6.35377 13.2988 6.28723 13.5406 6.22068 13.7824C3.32604 13.4715 1.9619 10.7427 1.89536 10.639C1.66246 10.1209 1.06357 9.91364 0.56449 10.19C0.0654144 10.4318 -0.134216 11.0535 0.131958 11.5717C0.198501 11.7444 2.02845 15.4404 6.05432 15.8549V18.9637C6.05432 19.5509 6.48686 20 7.05248 20H12.875C13.4406 20 13.8732 19.5509 13.8732 18.9637V14.8532C13.8732 14.2314 13.7401 13.6442 13.5072 13.1261C16.2022 12.0553 17.9989 9.91364 17.9989 7.46114Z" fill="#505F79"/>
</svg>
```

Reply:
i did it as you told me but there is no change